ui/pages/first-time-flow/index.scss

```
@@ -36,23 +61,60 @@
   }
 
   &__create-back {
+    margin-top: 44px;
     margin-bottom: 16px;
   }
 
   &__header {
     @include H1;
 
     margin-bottom: 24px;
-    color: black;
+    color: var(--white);
+  }
+  &__inputWrap{
```

```suggestion
  &__input-wrap{
```
- To be consistent with the coding style

---

fixed in 61b20a179b653d94185890b9069c56a5f6a9dd34